src/search/FindReplace.js

```diff
@@ -672,7 +672,10 @@ define(function (require, exports, module) {
             state = getSearchState(cm),
             replaceText = findBar.getReplaceText();
 
-        if (all) {
+        if(all === null){
```

Comment:
Add one space after the `if` and one before the `{` to fix some JSLint warnings.